iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/Tools.java

```diff
@@ -15,22 +15,33 @@
 import java.io.IOException;
 import java.net.SocketException;
 import java.net.UnknownHostException;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Tools
 {
     private static final long RETRY_TIMEOUT_ON_NETWORK_FAILURE = 60 * 1000;
```

Comment:
nit - and not directly related to this PR: these var names should also specify the time unit; is it 60 * 1000 ms/ 60 * 1000 secs/ 60 * 1000 mins ....?

Reply:
Milliseconds, and sure, I can scrub through these real fast